README.md

```diff
@@ -150,6 +150,8 @@ several other libraries, including:
 
 * [`loom`]: A testing tool for concurrent Rust code.
 
+* [`salvo`]: A powerful web framework built with a simplified design.
```

Comment:
I am not sure Salvo could be listed here because of https://github.com/tokio-rs/tokio/pull/7808/changes#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R132-R133 - it is not maintained by the Tokio team.

Reply:
Okay, that's clear now.